### PR TITLE
add footer section

### PR DESCRIPTION
**Pull request by oxSaturn**
Closes https://github.com/Velocimeter/frontend/issues/197

CleanShot-uT8zJQzJ@2x

CleanShot-iF44FtfP@2x

---

### Detailed summary
- Updated theme colors in `light.js` and `tailwind.config.js`
- Updated accent color and border radius in `_app.tsx`
- Updated styling in `ssVests.tsx`, `ssVotes.tsx`, `ssRewards.tsx`, `LiquidityPairs.tsx`, `header.tsx`, `PageWrapper.tsx`, `layout.tsx`, and `navigation.tsx`
- Added footer section in `layout.tsx`
- Removed unused imports and files

> The following files were skipped due to too many changes: `Frontend-v1-Original/public/images/Layer_Zero_Logo_Black.svg`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: Frontend-v1-Original/components/common/PageWrapper.tsx [hunk 1]
```diff
@@ -7,14 +6,13 @@ interface Props {
...
   const { address } = useAccount();
   return (
-    <div className="relative mt-0 flex h-full w-full flex-col pt-8">
+    <div className="flex h-full w-full flex-shrink-0 flex-grow flex-col">
       {address ? (
         <div>{props.children}</div>
       ) : (
-        <Paper className="fixed top-0 flex h-full w-full flex-col flex-wrap items-center justify-center bg-appBackground p-12 text-center shadow-none after:absolute after:top-0 after:left-0 after:hidden after:h-full after:w-full after:bg-waves after:bg-cover after:bg-no-repeat max-lg:my-auto max-lg:mb-0 after:xs:block lg:h-[100vh] lg:w-full">
-          <div className="max-lg:mt-24"></div>
+        <div className="flex h-full w-full flex-grow flex-col items-center justify-center px-10 text-center shadow-none md:bg-transparent lg:w-full">
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/common/PageWrapper.tsx:13]: Get rid of `Paper` from MUI, and clean up some classes.

File: Frontend-v1-Original/components/header/header.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ function Header() {
                 </Badge>
               </IconButton>
             )}
-            <span className="md:fixed md:right-5 md:bottom-5 xl:static">
+            <span className="md:max-[950px]:fixed md:max-[950px]:right-5 md:max-[950px]:bottom-5">
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/header/header.tsx:73]: Still don't have enough space for connect button between 768 and 950.

File: Frontend-v1-Original/components/layout/layout.tsx [hunk 3]
```diff
@@ -61,6 +63,35 @@ export default function Layout({
             </div>
           </div>
         )}
+        <footer className="mx-auto flex gap-x-3 pt-20 pb-10 text-sm">
+          <Link href="/claim" className="text-cyan-700 hover:underline">
+            Claim
+          </Link>
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/layout/layout.tsx:69]: I've moved the `Claim` link to the footer to give more space to navbar as I think it's not a permanent link, we'll likely take it down once the task is done.

File: Frontend-v1-Original/tailwind.config.js [hunk 4]
```diff
@@ -50,6 +50,9 @@ module.exports = {
         error: "#ff4141",
         cyan: {
           DEFAULT: "#00E8CA",
+          700: "#00CFB3",
+          800: "#00A892",
+          900: "#00695B",
```
Comment (oxSaturn) [inline Frontend-v1-Original/tailwind.config.js:55]: Even though I introduced 3 more colors, only `700` is used at the moment.

And here's how I get those color values:

https://color.adobe.com/create/color-wheel

CleanShot 2023-07-07 at 13 04 52@2x

File: Frontend-v1-Original/components/layout/layout.tsx (hunk 3 above)
Comment (oxSaturn) [reply Frontend-v1-Original/components/layout/layout.tsx:69]: Currently it's just a list of simple links, we could refactor the styles when we have more.

File: Frontend-v1-Original/components/layout/layout.tsx [hunk 5]
```diff
@@ -61,6 +63,35 @@ export default function Layout({
...
+          {[
+            {
+              href: "https://www.scanto.io/",
+              text: "sCANTO",
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/layout/layout.tsx:73]: `sCanto` icon removed, just doesn't fit the footer great, lol.

File: Frontend-v1-Original/components/common/PageWrapper.tsx [hunk 6]
```diff
@@ -7,14 +6,13 @@ interface Props {
...
           {props.placeholder}
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/common/PageWrapper.tsx:14]: My target is to get rid of placeholder as well in near future. We shouldn't hide most information behind those placeholder on non-web3 enabled browsers. Better to minimize those blockage.

Comment (t0rbik) [reply Frontend-v1-Original/components/common/PageWrapper.tsx:14]: totally agree. we will need to do some refactoring to show dex data without user connected as well

**Comment by t0rbik** [comment]
Do you think it may look better if the footer links located at the edge? not center. First opinion was that it attract to much attention kinda. what are your thoughts?

**Comment by oxSaturn** [comment]
> Do you think it may look better if the footer links located at the edge? not center. First opinion was that it attract to much attention kinda. what are your thoughts?

Currently our page has many different alignment, for instance, our logo is left aligned, menus are center aligned while connect button is right aligned. It's sort of messy IMO.

So the reason of footer to be center aligned is to keep consistency with menus in navbar. As you can see, most contents on the page are center-aligned at the moment. If we make footer left or right aligned, I reckon the page would become more messy as both logo and connect button are sort of unique elements on the page, they have nothing in common with footer section.

Regarding your concern, we can use some darker colors for links in footer, or even apply a large margin-top to push it away from our content section, but users will have to scroll a little bit to find them.

CleanShot-EriPxyH4@2x

File: Frontend-v1-Original/components/layout/layout.tsx [hunk 7]
```diff
@@ -45,7 +45,7 @@ export default function Layout({
         )}
         <SnackbarController />
         {!chain?.unsupported ? (
-          <main className="flex flex-grow flex-col">{children}</main>
+          <main className="flex flex-grow flex-col sm:pt-10">{children}</main>
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/layout/layout.tsx:48]: The space between navigation and content is currently set separately by respective pages, better just use `main` to control to keep consistency.

File: Frontend-v1-Original/pages/_app.tsx [hunk 8]
```diff
@@ -75,9 +75,9 @@ export default function MyApp({
               <RainbowKitProvider
                 chains={chains}
                 theme={rainbowKitDarkTheme({
-                  accentColor: "rgb(0, 243, 203)",
+                  accentColor: "#00E8CA",
                   accentColorForeground: "#222222",
-                  borderRadius: "small",
+                  borderRadius: "medium",
```
Comment (oxSaturn) [inline Frontend-v1-Original/pages/_app.tsx:80]: We use `rounded-lg` for active menu item, which is 0.5rem, i.e., 8px. `medium` would equal to `8px` in rainbowkit.

File: Frontend-v1-Original/components/navigation/navigation.tsx [hunk 9]
```diff
@@ -52,9 +52,7 @@ function Navigation() {
       <Link
         href={"/" + link}
         className={`cursor-pointer select-none appearance-none rounded-lg px-[14px] py-2 text-sm capitalize no-underline outline-0 transition-colors lg:text-base ${
-          active === link
-            ? "bg-cyan text-black"
-            : "hover:bg-[hsla(0,0%,100%,.04)]"
+          active === link ? "bg-cyan text-black" : "hover:text-cyan"
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/navigation/navigation.tsx:55]: Reckon the current style when hovering is too subtle to notice.

Comment (oxSaturn) [reply Frontend-v1-Original/components/navigation/navigation.tsx:55]: Another option is to get rid of the background as well, as the style is sort of conflict with our connect button.

CleanShot-Ka51orgH@2x

File: Frontend-v1-Original/components/layout/layout.tsx [hunk 10]
```diff
@@ -63,34 +63,91 @@ export default function Layout({
...
+                key={index}
+                href={href}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-cyan-700 hover:underline"
+              >
+                {text}
+              </a>
+            ))}
+          </nav>
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/layout/layout.tsx:150]: CleanShot-ksDMckPA@2x

**Review by t0rbik — APPROVED** [review]

**Pull request merged**